### PR TITLE
displayPhone

### PR DESCRIPTION
**Pull request by lxc150896**
task: https://dev.framgia.com/issues/212544

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -111,23 +116,25 @@ public function search(SearchRequest $request)
...
+
             return response()->json([
                 'data' => '1',
             ]);
         } else {
             return response()->json([
                 'data' => '0',
             ]);
-        }
+        }
```
Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 138</br>Whitespace found at end of line

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -120,7 +125,7 @@ public function getPhone($phone)
                 'phone' => $phone,
             ];
             $this->user->update($id, $data);
-
+            $this->usermeta->updateDisplayPhone($id, $radio);
             return response()->json([
```
Comment (BunnyPi04) [inline app/Http/Controllers/HomeController.php:130]: thiếu dòng trắng trc return

File: app/Http/Controllers/HomeController.php [hunk 3]
```diff
@@ -120,11 +125,13 @@ public function getPhone($phone)
                 'phone' => $phone,
             ];
             $this->user->update($id, $data);
+            $this->usermeta->updateDisplayPhone($id, $radio);
 
             return response()->json([
                 'data' => '1',
             ]);
         } else {
```
Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 133</br>Blank line found at start of control structure

Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 133</br>Blank line found at start of control structure

**Pull request merged**